src/ripple/protocol/impl/LedgerFormats.cpp

```diff
@@ -267,6 +267,21 @@ LedgerFormats::LedgerFormats()
             {sfPreviousTxnLgrSeq,    soeREQUIRED}
         },
         commonFields);
+
+    add(jss::CFTokenIssuance,
+        ltCFTOKEN_ISSUANCE,
+        {
+            {sfIssuer,               soeREQUIRED},
+            {sfTransferFee,          soeREQUIRED},
+            {sfOwnerNode,            soeREQUIRED},
+            {sfAssetCode,            soeREQUIRED},
+            {sfAssetScale,           soeREQUIRED},
+            {sfMaximumAmount,        soeREQUIRED},
+            {sfOutstandingAmount,    soeREQUIRED},
+            {sfLockedAmount,         soeREQUIRED},
+            {sfCFTokenMetadata,      soeOPTIONAL},
+        },
+        commonFields);
```

Comment:
`sfAssetScale`, `sfOutstandingAmount`, `sfLockedAmount`, and `sfTransferFee` can all be `soeDEFAULT` if you want. I believe the upside is that you save a tiny bit of storage by not serializing those fields if they're `0`, but I could be wrong.

Reply:
I agree with both comments above -- one nice fallout of that is that the spec doesn't need to be changed (though the implementation in this PR does).